src/gui/symbology/qgssymbolselectordialog.h

```diff
@@ -127,21 +127,21 @@ class GUI_EXPORT QgsSymbolSelectorWidget: public QgsPanelWidget, private Ui::Qgs
      */
     QgsSymbol *symbol() { return mSymbol; }
 
+    /**
+     * Load the given symbol into the widget.
+     * \param symbol The symbol to load.
+     * \param parent The parent symbol layer item. If the parent parameter is null, the whole symbol and model will be reset.
+     * \note not available in Python bindings
+     */
+    void loadSymbol( QgsSymbol *symbol, SymbolLayerItem *parent = nullptr ) SIP_SKIP;
```

Comment:
Can you add a TODO QGIS 4 note here saying "QGIS 4: transfer ownership of symbol", and in the dox note that ownership is NOT transferred and that the symbol MUST exist for the lifetime of the widget

Reply:
Done.